test/torchaudio_unittest/datasets/vctk_test.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from torchaudio.datasets import vctk
+
+from torchaudio_unittest.common_utils import (
+    TempDirMixin,
+    TorchaudioTestCase,
+    get_whitenoise,
+    save_wav,
+    normalize_wav,
+)
+
+
+# Used to generate a unique utterance for each dummy audio file
+NUMBERS = [
```

Comment:
This is not a part of VCTK. You can use some of the actual utterances from the dataset.

Reply:
The updated VCTK from PR #812 uses .flac format. Should I use .wav instead?

Reply:
Update vctk_test.py to use actual utterances from the dataset and mimic the original data structure. However, the dataset VCTK-Corpus-0.92.zip contains the update data format in .flac form. The original vctk.py before PR #812 uses .wav format. I suppose we should .flac format?